Remove WildFly distribution from documentation (#1666)
keycloak/keycloak-documentation: server_development/topics/providers.adoc

org.acme.provider.MyThemeSelectorProviderFactory
----

> You can configure your provider through server configuring.

Reviewer: ```suggestion
To configure your provider, see the (https://www.keycloak.org/server/configuration-provider)[Configuring Providers] guide.
```

> For example by adding starting the server with the following arguments:

Reviewer: ```suggestion
For example, to configure a provider you can set options as follows:
```

> Hence there can be only single implementation of this provider active for the {project_name} server. If there are multiple provider implementations available to the server runtime,

Reviewer: We can probably remove all the text related to configuring a default provider because it is documented at https://www.keycloak.org/server/configuration-provider.

Author: Focus here is to remove the WildFly documentation, not to enhance the Quarkus documentation. So this should be handled in a follow-up.

particular provider factory implementation. Some provider types can be retrieved with the usage of `ComponentModel` as the second argument and some (for example `Authenticator`) even
need to be retrieved with the usage of `KeycloakSessionFactory`. It is not recommended to implement your own providers this way as it may be deprecated in the future.

> === Registering provider implementations

Reviewer: We can remove or reference https://www.keycloak.org/server/configuration-provider.

Author: Focus here is to remove the WildFly documentation, not to enhance the Quarkus documentation. So this should be handled in a follow-up.
